courses/machine_learning/deepdive2/feature_engineering/solutions/Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2721 @@
+{
```

Comment:
<p>Lak mentioned that Learning Objectives should be lab specific, so "Setup up the environment" can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Agreed.  Removed.</p><p> </p>